docs/deployment/uninstallation.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Uninstallation"
+layout: default
+sort: 3
+---
+
+# Uninstallation
+{: .no_toc}
+
+## Table of contents
+{: .no_toc .text-delta}
+
+1. TOC
+{:toc}
+
+---
+
+## Uninstallation
+
+If you followed the deployment instructions above you can simply do:
```

Comment:
Where are these deployment instructions? Maybe we should have combine this md file with that?